index.html

```diff
@@ -1,26 +1,124 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" xmlns="http://www.w3.org/1999/html">
 <head>
     <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles/Styles.css">
+
     <title>Title</title>
```

Comment:
Ha Chaudhry,

Ik kon op de laatste commit in de bovenste regels geen commentaar geven. Vandaar deze opmerking in een eerder stadium. Ik zal trachten om de code na te lopen. Morgen nog een laatste hand aan mij eigen code zodat ik deze woensdag kan uitleveren.

Je bent vergeten een pagina titel toe te voegen aan deze landingspagina;

Ik heb zelf bv Rekute digital agency for new experiences.